app/views/portfolio_evidence_mailer/portfolio_failed.html.erb

```diff
@@ -47,6 +47,11 @@
   <main>
     <h1> <%= @doubtfire_product_name %> Notification </h1>
     <p>Hi <%= @student.first_name %>,</p>
+    <% if @project.portfolio_auto_generated %>
+      <p>
+        Profile generation has been automatically triggered because you have not yet generated it beyond the date that the unit chair has specified.
```

Comment:
We can probably make this more friendly... "Profile generation has been automatically triggered. You can customise your portfolio and recreate up until the due date indicated by the teaching staff."